jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/ctl/ActionSleep.java

```diff
@@ -32,6 +32,8 @@
 /** A task that kicks off a asynchronous operation that simply waits and exits.  For testing. */
 public class ActionSleep extends ActionCtl /* Not ActionAsyncTask - that is a container-item based. */
 {
+    private static int MaxSleepMillis = 20*1000;
```

Comment:
`maxSleepMillis`?

Reply:
Logically, it's a constant. I'll add `final`.  ALL_CAPS_IS_A_BIT_SHOUTY :-)